cli/src/potpie/api_wrapper.py

```
@@ -0,0 +1,137 @@
+import json
+import logging
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)